storages/indexeddb-storage/Cargo.toml

```diff
@@ -0,0 +1,27 @@
+[package]
+name = "gluesql_indexeddb_storage"
+version = "0.13.0"
+edition = "2021"
+authors = ["Taehoon Moon <taehoon.moon@outlook.com>"]
```

Comment:
you can become the first author in here.

ref.
https://github.com/gluesql/gluesql/blob/9950ee6f43fa73f06a5d53ea74d200586c93f8cb/storages/shared-memory-storage/Cargo.toml#L5-L8